contracts/mainnet/connectors/morpho-blue/events.sol

```diff
@@ -0,0 +1,105 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+import {Id, MarketParams} from "./interfaces/IMorpho.sol";
+
+contract Events {
+    event LogSupplyAssets(
+        Id indexed id,
+        uint256 assets,
+        uint256 shares,
+        uint256 getId,
+        uint256 setId
+    );
+
+    event LogSupplyOnBehalf(
+        Id indexed id,
+        uint256 assets,
+        uint256 shares,
+        address onBehalf,
+        uint256 getId,
+        uint256 setId
+    );
```

Comment:
Following: https://github.com/Instadapp/dsa-connectors/pull/321#discussion_r1428848661

You could also index `onBehalf` and `receiver` whenever it makes sense to you
I'm also pointing out that you do not log `caller = msg.sender` but it may be a design choice from you!

Reply:
I've resolved most of these comments already. 

Indexed both onBehalf and receiver on the events.